### ISSUE TITLE
Support for bigbed

### ISSUE THREAD

**Issue by pkerpedjiev** [jackh726/bigtools#50]
Hi!

I'm trying to open [this bigBed](https://genomeark.s3.amazonaws.com/species/Pan_troglodytes/mPanTro3/assembly_curated/gene/mPanTro3_refGene.bb) file and running into the following issue:

```
>>> import pybigtools
>>> f = open('mPanTro3_refGene.bb', 'rb')
>>> x = pybigtools.open(f)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
pybigtools.BBIReadError: File-like object is not a bigWig or bigBed. Or there was just a problem reading: File is not a bigBed.
```

I was able to open it without problems using pyBigWig.

Is this expected? `bigtools` supports bigbed, right?

**Comment by jackh726** [comment]
Let me look. This should work.

**Comment by jackh726** [comment]
This should be fixed in pybigtools 0.2.1. Please let me know if you still run into issues!
